### PR TITLE
BuildNumber on Experimental packages

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Merging to fix the build.

### CODE REVIEW

**Comment by natemcmaster** [comment]
hold up...this still isn't right. I suggest you copy the same logic we use for normal packages, and just slap "experimental" in front of it

**Comment by natemcmaster** [comment]
```xml

<Project>
  <PropertyGroup>
    <VersionPrefix>2.1.0</VersionPrefix>
    <VersionSuffix>preview1</VersionSuffix>
    <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
    <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' != 'rtm' ">$(VersionPrefix)-$(VersionSuffix)-final</PackageVersion>
    <BuildNumber Condition="'$(BuildNumber)' == ''">t000</BuildNumber>
    <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>

    <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix>
    <ExperimentalVersionSuffix>alpha1</ExperimentalVersionSuffix>
    <ExperimentalPackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(ExperimentalVersionSuffix)' == 'rtm' ">$(ExperimentalVersionPrefix)</ExperimentalPackageVersion>
    <ExperimentalPackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(ExperimentalVersionSuffix)' != 'rtm' ">$(ExperimentalVersionPrefix)-$(VersionSuffix)-final</ExperimentalPackageVersion>
    <ExperimentalVersionSuffix Condition="'$(ExperimentalVersionSuffix)' != '' And '$(BuildNumber)' != ''">$(ExperimentalVersionSuffix)-$(BuildNumber)</ExperimentalVersionSuffix>
  </PropertyGroup>
</Project>

**Comment by ryanbrandenburg** [comment]
:up::date:

File: version.props [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-    <ExperimentalVersionPrefix>0.5.0</ExperimentalVersionPrefix>
-    <ExperimentalVersionSuffix>alpha1</ExperimentalVersionSuffix>
-    <ExperimentalPackageVersion Condition="'$(IsFinalBuild)' == 'true'">$(ExperimentalVersionPrefix)-$(ExperimentalVersionSuffix)-final</ExperimentalPackageVersion>
     <VersionSuffix>preview1</VersionSuffix>
     <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' == 'rtm' ">$(VersionPrefix)</PackageVersion>
     <PackageVersion Condition="'$(IsFinalBuild)' == 'true' AND '$(VersionSuffix)' != 'rtm' ">$(VersionPrefix)-$(VersionSuffix)-final</PackageVersion>
     <BuildNumber Condition="'$(BuildNumber)' == ''">t000</BuildNumber>
     <VersionSuffix Condition="'$(VersionSuffix)' != '' And '$(BuildNumber)' != ''">$(VersionSuffix)-$(BuildNumber)</VersionSuffix>
+
+    <ExperimentalVersionPrefix>0.4.0</ExperimentalVersionPrefix>
```
Comment (natemcmaster) [inline version.props]: my bad. should be 0.5.0. Other than that, :shipit:

**Pull request merged**